README.rst

```diff
@@ -13,3 +16,37 @@ Versioning is done in the following manner:
 All stack versions are listed correspond to a Docker image listed at:
 
 - https://hub.docker.com/r/scrapinghub/scrapinghub-stack-hworker/tags/
+
+Release procedure
+=================
+
+When you're going to release a new version of the stack, you should:
+
+1. Do not forget to pull latest changes from master::
+
+    git pull origin
+
+2. Tag it with a correct tag string (see Versioning section above)::
+
+    git tag <tag>
+
+  Example::
+
+    git tag 20160731
+
+3. Push tags to the repo::
+
+    git push -f --tags
```

Comment:
btw, `--force` is not needed because we are releasing a new tag, not updating an existent tag.